examples/stupid/stupid01-basic.py

```diff
@@ -1,43 +1,57 @@
 import random
-import math
 from agents import Agent, Model, run
 
 
 class Bug(Agent):
     def setup(self, model):
         self.size = 8
         self.color = (255, 0, 0)
+
+        # Align to center of tile
         self.align()
+
+        # Optimization, to allow quick lookup of which agents are on a tile
+        # can we hide this somehow?
         self.update_current_tile()
```

Comment:
`update_current_tile` should probably just be called by the model when calling `add_agent`.

Reply:
yeah, let's do that

Reply:
Turns out the model actually already does that, so this is probably just a leftover from an earlier version.